test/metabase/test.clj

```diff
@@ -319,12 +318,11 @@
                                      (t2/update! (t2/table-name User) {:id [:in existing-admin-ids]} {:is_superuser false}))
         temp-admin                 (first (t2/insert-returning-instances! User (merge (with-temp-defaults User)
                                                                                       attributes
-                                                                                      {:is_superuser true})))
-        primary-key                (mdb.u/primary-key User)]
+                                                                                      {:is_superuser true})))]
     (try
       (thunk temp-admin)
       (finally
-        (t2/delete! User primary-key (primary-key temp-admin))
+        (t2/delete! User :id (:id temp-admin))
```

Comment:
You don't even need `:id` here, you can just do `(t2/delete! <model> <id>)` now.

Reply:
right, removed it in https://github.com/metabase/metabase/pull/31682/commits/9fb4184bacb8a97e44c5f1ac3347a5ca6cc92169